test/operator.test.js

```diff
@@ -305,7 +305,41 @@ contract('SelfToken', function (accounts) {
   })
 
   it("should not be able to unauthorize an authorized official operator when accepting all official operators", async function () {
-    await selfToken.revokeOperator(operator1.address);
+    await shouldFail.reverting(selfToken.revokeOperator(operator1.address, {
+      from: user1
+    }))
```

Comment:
What's the state of operator1 before this?
The success and failure of revoking an operator is not related to whether the operator is an official operator.
The precise description could be
"should be able to revoke operator1 if it has been authorized by user1, but since operator1 is an official operator and user1 accepts all official operators, so operator1 is still a valid operator for user1"

Reply:
change it description